packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -195,36 +198,33 @@ export class VaAccordion {
       'usa-accordion': true,
     });
 
-    const accordionItemIDs = [...this.el.children]
-      .filter((el) => el.tagName.toLowerCase() === 'va-accordion-item')
-      .map((el) => el.id);
-    
```

Comment:
This block was no longer needed after the `aria-controls` attribute was removed. TypeScript was complaining so I opted to remove it.